spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -5,19 +5,230 @@
 module PuppetX
   module PowerShell
     class PowerShellManager; end
+    if Puppet::Util::Platform.windows?
+      module WindowsAPI
+        require 'ffi'
+        extend FFI::Library
+
+        # TODO: necessary to find CRT version or just "msvcrt"
+        def self.msvcrt_name
+          host_part = RbConfig::CONFIG['host_os'].split("_")[1]
+          manifest  = File.join(RbConfig::CONFIG['bindir'], 'ruby.exe.manifest')
+
+          if host_part && host_part.to_i > 80 && File.exists?(manifest)
+            "msvcr#{host_part}"
+          else
+            "msvcrt"
```

Comment:
I think just `msvcrt` is sufficient here - verify for certain to simplify this code.